### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by fandangos**
#### About pull request
- This pull request adds Brazilian Portuguese translation as an option to the theme.

#### What's changed to accomplish [problem / feature] described in issue
This pull request doesn't change any previews code, except adding pt (portuguese) as an option to the main.conf file. 
All the translation is nested in pt folder. 
I have not translated the William Shakespeare quotes and other examples because those will end up being replaced by other users when using this theme, so I left those in place. 

I also translated the README file and named it _README-pt.md (as the same as the Japanese one) and this can be added to the main project page by the main author of the theme. Because this is the author's choice, I have not added a link to the brazilian portuguese README.md file since those are linked directly to the author's of this theme github page. 

Please, considere this as a gratitude for your work since I'm using your jekyll theme on my personal blog since yesterday (https://fandangos.github.io) but with only English and Portuguese language options as I don't  speak Japanese.

### CODE REVIEW

**Comment by MrGreensWorkshop** [comment]
I can't tell you how I feel right now. I'm so happy. Please give me a little bit time to review.

**Comment by fandangos** [comment]
I forgot to translate 2 titles in the README, I'm sorry for that. 
I've added a new commit and now it should be fully translated.

**Comment by MrGreensWorkshop** [comment]
Thanks to you, when I was checking the PR, I found a bug,  #120 will fix your website month translation.
There are small things needs to change. Still checking.

**Comment by MrGreensWorkshop** [comment]
I make some changes, Could you please take a look to this commit d85467ea

**Review by MrGreensWorkshop — COMMENTED** [review]
Marked pages are good to go.

**Comment by fandangos** [comment]
I made some small changes to improve the translation.
In portuguese, you can use some words in English and those will be left in the phrase like the original. 
If you translate Theme to Tema, it would be spoken before the title of the theme: Tema Mr. Green.
That's why I changed it.

Everything looks really good and I really appreciate the credits. 
I'll write about this colaboration on my blog that is powered by Mr Green :))

**Comment by MrGreensWorkshop** [comment]
Everything is look good. Forgive me for my poor translations. I will update demo page too. I appreciate your time. Thank you.

File: README-pt.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+[[English](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README.md#readme)] [[日本語](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README-jp.md#readme)] [Português do Brasil]
+
+## Mr. Green Jekyll Tema
```
Comment (MrGreensWorkshop) [inline README-pt.md:3]: @fandangos  title of readme file is "Mr. Green Jekyll Tema" but line 145 is Tema Mr. Green Jekyll, is it ok like this? 
ot should this line better with "Tema Mr. Green Jekyll" ?

Comment (fandangos) [reply README-pt.md:3]: It's more pleasing to the ears "Tema Mr. Green Jekyll"

**Review by MrGreensWorkshop — APPROVED** [review]
well all done.

**Pull request merged**